src/app/App.tsx

```diff
@@ -108,6 +124,8 @@ export class App extends React.Component<Props> {
     store.dispatch(appMounted())
   }
 
+  dynamicLinksRemoveListener: (() => void) | undefined
```

Comment:
would it be clearer to have this as something like
```
dynamicLinksRemoveListener() {
  if (FIREBASE_ENABLED) {
    ...
  }
}
```
instead of having this undefined and assigned in a different part of the code? would also remove the need for the optional chaining on line 151

Reply:
This line is only the type definition, not the implementation. The implementation is attached [here](https://github.com/valora-inc/wallet/pull/2886/files#diff-7c79023b16fd616788af0fdcb31d619ae04da7259aa564fd72fb0ea07d8ebe07R104), which is already wrapped in the `FIREBASE_ENABLED` check.